src/webapp/components/survey-list/SurveyList.tsx

```diff
@@ -82,10 +82,18 @@ export const SurveyList: React.FC<SurveyListProps> = ({ surveyFormType }) => {
                             surveys
                         ) && (
                             <ButtonWrapper>
-                                {surveyFormType === "PPSPatientRegister" && (
+                                {["PPSPatientRegister", "PrevalenceCaseReportForm"].includes(
```

Comment:
could you please create an array  like SURVEYS_WITH_FILTER_ENABLED or something similar and use it everywhere.